be/src/exec/pipeline/olap_chunk_source.cpp

```diff
@@ -357,7 +357,7 @@ Status OlapChunkSource::buffer_next_batch_chunks_blocking_for_workgroup(size_t b
 
             ChunkUniquePtr chunk(
                     ChunkHelper::new_chunk_pooled(_prj_iter->encoded_schema(), _runtime_state->chunk_size(), true));
-            _status = _read_chunk_from_storage(_runtime_state, chunk.get());
```

Comment:
this code should not been try catched now, it may use the expr framework and some code of storage layer